docs/README.md

```diff
@@ -40,96 +33,57 @@ and considered to be a "beta" quality release. We expect significant
 modifications will be made prior to its general availability, it is not
```

Comment:
Not related to your change, but we probably want to rewrite or remove this paragraph for GA.